src/components/icons/DefaultToken.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+
+interface DefaultTokenProps {
+  size?: number;
+  symbol?: string;
+}
+
+export default function DefaultToken({
+  size,
+  symbol,
+}: DefaultTokenProps): JSX.Element {
+  // const TextElement = Text as any;
```

Comment:
what's this for?

Reply:
removed unused comment, comment was use as comparison before not needed now have removed it